docs/arguments.rst

```diff
@@ -66,8 +66,83 @@ threads
     ``4``
 
 trusted_proxy
-    IP address of a client allowed to override ``url_scheme`` via the
-    ``X_FORWARDED_PROTO`` header.
+    IP address of a remote peer allowed to override various WSGI environment
+    variables using proxy headers.
+
+    For unix sockets, set this value to ``localhost`` instead of an IP address
+
+    default ``None``
```

Comment:
the `default` pattern used here is different from other settings in this file. I'd probably reformat all of them to a separate line like

```
Default: ``None``
```